lib/Google.js

```diff
@@ -107,9 +107,9 @@
 		var url = GoogleMapsLoader.URL;
 
 		url += '?callback=' + GoogleMapsLoader.WINDOW_CALLBACK_NAME;
-
-		url += '&sensor=' + ((GoogleMapsLoader.SENSOR === true || GoogleMapsLoader.SENSOR === 'true') ? 'true' : 'false');
-
+		if(GoogleMapsLoader.SENSOR) {
```

Comment:
Can you please add this variable into other properties? Just like in this [pull request](https://github.com/Carrooi/Js-GoogleMapsLoader/pull/28/files#diff-faf7c0808229ee587378c07a22d69f84R53).

Also missing white space after `if`